lib/src/admin.dart

```diff
@@ -48,27 +50,21 @@ class FirebaseAdmin {
   ///       privateKey: 'your-private-key',
   ///     );
   ///     var app = FirebaseAdmin.instance.initializeApp(
-  ///       credential: certificate,
-  ///       databaseURL: 'https://your-database.firebase.io',
+  ///       new AppOptions(
+  ///         credential: certificate,
+  ///         databaseURL: 'https://your-database.firebase.io')
   ///     );
   ///
   /// See also:
   ///   * [App]
   ///   * [cert]
   ///   * [certFromPath]
-  App initializeApp({
-    @required js.Credential credential,
-    @required String databaseURL,
-    String name: js.defaultAppName,
-  }) {
+  App initializeApp(@required js.AppOptions options, [String name]) {
```

Comment:
I think `@required` annotation in this case is a bit redundant since we moved away from named arguments and user is required to pass a value for `options` in any case.

Instead I'd add `assert(options != null);` as the first line in `initializeApp`.